src/alttxt/generator.py

```diff
@@ -28,34 +29,83 @@ def __init__(
         - grammar: The grammar model to use for generating the description
         """
         self.descriptions: "dict[str, Any]" = phrases.DESCRIPTIONS
-        self.verbosity: Verbosity = verbosity
         self.level: Level = level
-        self.explain: Explanation = explain
+        self.structure: bool = structured
```

Comment:
```suggestion
        self.structured: bool = structured
```

Reply:
This will need a few changes in this file to make all usage of this variable match the new name